src/main/java/service/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface DataConverter {
+    List<FruitTransaction> convertToTransaction(List<String> strings);
```

Comment:
```suggestion
    List<FruitTransaction> convertToTransactions(List<String> lines);
```
be careful with naming
don't name "container" types with their classname - like String string, List list etc, it tells us nothing about what it actually stores